mbus-api/src/message_bus/v0.rs

```diff
@@ -63,6 +63,12 @@ pub trait MessageBusTrait: Sized {
         only_one!(nodes)
     }
 
+    /// Generic JSON gRPC call
+    #[tracing::instrument(level = "debug", err)]
+    async fn json_grpc_call(request: JsonGrpcRequest) -> BusResult<String> {
```

Comment:
maybe move this to the top or bottom of the trait to keep the node/pool/nexus/volume together?

Reply:
Done - moved to bottom.